### PR TITLE
Solidus support

### PR DESCRIPTION
**Pull request by alexblackie**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,8 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
+  - 2.1.10
+  - 2.2.5
+  - 2.3.1
```
Comment (jhawthorn) [inline .travis.yml]: One ruby version is sufficient. We'll probably want to add different solidus versions and testing against all versions of both is excessive.

File: Gemfile [hunk 2]
```diff
@@ -1,3 +1,3 @@
 source 'https://rubygems.org/'
-gem 'spree', github: 'spree/spree', branch: '2-4-stable'
```
Comment (jhawthorn) [inline Gemfile]: Probably should refer to solidus here. Ideally [in the way we do elsewhere](https://github.com/solidusio/solidus_auth_devise/blob/master/Gemfile#L3-L4)

File: solidus_static_content.gemspec [hunk 3]
```diff
@@ -18,19 +18,19 @@ Gem::Specification.new do |s|
...
+  s.add_development_dependency 'capybara', '~> 2.7'
   s.add_development_dependency 'factory_girl', '~> 4.4'
   s.add_development_dependency 'ffaker'
   s.add_development_dependency 'database_cleaner', '1.2.0'
-  s.add_development_dependency 'rspec-rails',  '~> 2.14'
+  s.add_development_dependency 'rspec-rails',  '~> 3.5'
   s.add_development_dependency 'sqlite3', '~> 1.3.8'
-  s.add_development_dependency 'selenium-webdriver'
-  s.add_development_dependency 'poltergeist', '~> 1.5.0'
+  s.add_development_dependency 'poltergeist', '~> 1.10.0'
```
Comment (jhawthorn) [inline solidus_static_content.gemspec]: can probably use semver `s.add_development_dependency 'poltergeist', '~> 1.10'`

File: solidus_static_content.gemspec [hunk 4]
```diff
@@ -18,19 +18,19 @@ Gem::Specification.new do |s|
...
   s.add_development_dependency 'simplecov', '~> 0.7.1'
   s.add_development_dependency 'coffee-rails', '~> 4.0.0'
-  s.add_development_dependency 'sass-rails', '~> 4.0.0'
-  s.add_development_dependency 'guard-rspec'
+  s.add_development_dependency 'sass-rails', '~> 5.0.2'
```
Comment (jhawthorn) [inline solidus_static_content.gemspec]: Do we need to specify sass-rails at all?

Comment (alexblackie) [inline solidus_static_content.gemspec]: I removed it and `coffee-rails` and nothing broke so I assume not.

**Comment by jhawthorn** [comment]
:+1:

**Pull request merged**